IFC4x3/Properties/c/CurrentRMS_0Y55X3X$b2LR4CS5G0nLXs/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CurrentRMS_0Y55X3Xb2LR4CS5G0nLXs" Name="CurrentRMS" UniqueId="22145843-87f9-4255-b10c-705400c55876" PrimaryDataType="IfcElectricCurrentMeasure" />
```

Comment:
Do we not have some sort of a similar property available?

Reply:
I didn't find equivalent property.